marlowe-playground-client/src/Wallet.purs

```diff
@@ -1115,24 +1115,32 @@ inputItem isEnabled person (DepositInput accountId party token value) =
 inputItem isEnabled person (ChoiceInput choiceId@(ChoiceId choiceName choiceOwner) bounds chosenNum) =
   li
     [ classes [ aHorizontal, ClassName "flex-wrap", ClassName "choice-row" ] ]
-    [ div []
-        [ p [ class_ (ClassName "choice-input") ]
-            [ spanText "Choice "
-            , b_ [ spanText (show choiceName) ]
-            , spanText ": Choose value "
-            , marloweActionInput isEnabled (SetChoice choiceId) chosenNum
-            ]
-        , p [ class_ (ClassName "choice-error") ] error
-        ]
-    , button
-        [ classes [ plusBtn, smallBtn, if (isEnabled && inBounds chosenNum bounds) then (ClassName mempty) else Classes.hide ]
-        , enabled (isEnabled && inBounds chosenNum bounds)
-        , onClick $ const $ Just
-            $ AddInput (Just person) (IChoice (ChoiceId choiceName choiceOwner) chosenNum) bounds
-        ]
-        [ text "+" ]
-    ]
+    ( [ div []
+          [ p [ class_ (ClassName "choice-input") ]
+              [ spanText "Choice "
+              , b_ [ spanText (show choiceName <> ":") ]
+              , br_
+              , spanText "Choose value "
+              , marloweActionInput isEnabled (SetChoice choiceId) chosenNum
+              ]
+          , p [ class_ (ClassName "choice-error") ] error
+          ]
```

Comment:
I think it is nicer to disable the button than to make it disappear, otherwise the user does not have a hint that if he fixes something he will be able to click it

Reply:
The problem is that it makes layout much harder because we are displaying a warning message that covers almost the whole width. Also I would be happy for a better solution but I found it very tricky to make the wrapping look good, getting rid of it like this was _much_ easier

Reply:
I'm going to leave this as it is in the PR because I'm really sick of css 😞 but I would be very open to someone coming up with something better. This at least fixes a layout bug.